lib/src/mt_ptp.c

```diff
@@ -20,6 +20,10 @@
 
 #define MT_PTP_EBU_SYNC_MS (10)
 
+#define MT_PTP_USE_PI
```

Comment:
Can we use a bool var in "struct mt_ptp_impl* ptp" instead of a build marco?

Reply:
no problem